doc/Type/Metamodel/PrivateMethodContainer.pod6

```diff
@@ -42,6 +42,25 @@ Adds a private method C<$code> with name C<$name>.
 
 Returns a hash of C«name => &method_object»
 
+=head2 method private_methods
```

Comment:
This probably belongs in a different PR. It's pretty good, and not worth the while to hold it while we solve the other one.